basic-linux-keysniffer/io/consts.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by gnir on 6/7/19.
+//
+
+#ifndef BASIC_LINUX_KEYSNIFFER_CONSTS_H
+#define BASIC_LINUX_KEYSNIFFER_CONSTS_H
+
+namespace KeyboardEventReader {
+
+    /**
+     * An enum class containing the input event values corresponding to the key down
+     * and key hold event.
+     * This enum is used in order to determine which events are relevant to the key sniffer
+     * and which are not.
+     */
+    enum class KeyboardEventTypes {
+        DOWN = 1,
```

Comment:
For enum classes, the convention (in most of the them, at least) is to use normal variable casing.
You write in all uppercase only if it's a regular enum.